test/Microsoft.CodeAnalysis.Razor.Test/DefaultTagHelperDescriptorFactoryTest.cs

```diff
@@ -1306,7 +1306,7 @@ public class DynamicTestTagHelper : {typeof(AspNetCore.Razor.TagHelpers.TagHelpe
         {{
         }}";
             var syntaxTree = CSharpSyntaxTree.ParseText(text);
-            var compilation = TestCompilation.Create(syntaxTree);
+            var compilation = TestCompilation.Create(typeof(DefaultTagHelperDescriptorFactoryTest), syntaxTree);
```

Comment:
Think about making this into a method, it's not super create just looking at the code of all the tests what this does. It would be better to pull it out into a method and make it clear what it means.